api/tests/src/api/competition_alpha/test_competition_route_get.py

```diff
@@ -34,6 +35,66 @@ def test_competition_get_200_with_api_key(client, api_auth_token, enable_factory
     assert response_competition["open_to_applicants"] == competition.open_to_applicants
 
 
+def test_competition_get_with_instructions_200(client, api_auth_token, enable_factory_create):
+    # Create a competition with instructions
+    competition = CompetitionFactory.create(with_instruction=True)
+
+    # Mock the download_path property for CompetitionInstruction
+    presigned_url = "https://example.com/competition-instructions/file.pdf"
+    with mock.patch(
+        "src.db.models.competition_models.CompetitionInstruction.download_path",
+        new_callable=mock.PropertyMock,
+        return_value=presigned_url,
+    ):
+        resp = client.get(
+            f"/alpha/competitions/{competition.competition_id}", headers={"X-Auth": api_auth_token}
+        )
```

Comment:
Instead of patching this - could we just rely on the actual logic / check the file exists?

With how the factory is setup, the instruction object should be downloadable without any additional work, we might pass in file_contents to the competition instruction factory and check the contents of the file we download to be safe.